### PR TITLE
Adds filters and search to orders and attendees page in Ticket sectio…

### PR DESCRIPTION
**Pull request by Princu7**
…n of event dashboard. Fixes the issue #3106.
Test Server is Here
https://ancient-thicket-44821.herokuapp.com/events/8/tickets/
Login Credentials:
```
email-id : princu7@gmail.com
password: fossasia
```

* Add the date and ticket filter on the attendee page. Both the filters can work simultaneously and removing or adding a filter doesn't affect the value of the other filter.
* Adds the search box on the orders page

Screenshots:
Attendee page
screenshot from 2017-02-11 19-23-55
screenshot from 2017-02-11 19-24-54

Orders Page
screenshot from 2017-02-11 19-27-09

@SaptakS @niranjan94 @mariobehling  Kindly review. Thanks

### CODE REVIEW

**Comment by Princu7** [comment]
@SaptakS Fixed the codacy issues :)

**Comment by niranjan94** [comment]
@Princu7 `attendees.js` and `orders.js` seems to have some similar code. See if you're able to take common things out. Also fix codacy erros.

**Comment by Princu7** [comment]
@niranjan Thanks so much for the review :) I eliminated the redundancy but I need some guidance on the last codacy issue. I don't understand what it is trying to say and how to solve it. Please assist me. Thanks!!

**Comment by niranjan94** [comment]
@Princu7 codacy gives every function/file a complexity rating based on the number of _paths_ that get split up inside a method. For example, in a simple if/else statement, we have two different paths. An if/elseif/else has 3. 

Complexity errors can be just considered as warning. If possible, you could try to simply conditional statements. You could also use ternary operators for simple conditional assignments. If not possible, just ignore and add a comment below the line that it cannot be resolved.

File: app/static/js/events/ticket_sales/attendees.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+var $ticketFilterForm = $("#ticket-filter-form");
+var base_url = [location.protocol, '//', location.host, location.pathname].join('');
+
+$(".ticket-filter-open").click(function() {
+    $ticketFilterForm.show();
+    $ticketFilterForm.find("select").val("");
+});
+
+function make_request() {
```
Comment (Princu7) [inline app/static/js/events/ticket_sales/attendees.js]: @niranjan94 I used conditional statements this time but the issue still persists. Maybe we can ignore it. There is not much to worry about. Please review it. Thanks!!

**Pull request merged**